test/parallel/test-crypto-hmac.js

```diff
@@ -8,6 +8,14 @@ if (!common.hasCrypto) {
 }
 const crypto = require('crypto');
 
+// Test for binding layer robustness
+const binding = process.binding('crypto');
+(() => {
+  const h = new binding.Hmac();
+  // Fail to init the Hmac with an algorithm.
+  assert.throws(() => h.update('hello'), TypeError);
```

Comment:
We've been moving toward using a regular expression as the final argument to `assert.throws()`. It would look something like `/^TypeError: whatever the error message is$/`.